app/javascript/dashboard/modules/search/components/SearchResultContactItem.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <router-link :to="navigateTo" class="contact-item">
+    <woot-thumbnail :src="thumbnail" :username="name" size="24px" />
+    <div class="contact-details">
+      <h5 class="text-block-title name">{{ name }}</h5>
+      <p class="details-meta">
+        <span v-if="email" class="email">{{ email }}</span>
+        <span v-if="phone" class="separator">·</span>
```

Comment:
@nithindavid Instead of this symbol `·`, what about using this symbol `•`, feels more visible. Thoughts?